4food/src/containers/Router/index.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { ConnectedRouter } from "connected-react-router";
+import { Switch, Route } from "react-router-dom";
+import Home from "../Home";
+import Login from "../Login";
+import Profile from "../Profile";
+import RestaurantPage from "../RestaurantPage"
+import FeedPage from "../FeedPage"
+import SignUp from "../SignUp";
+import Address from "../Address";
+import Cart from "../Cart"
+
+export const routes = {
+  Home: "/",
+  Login: "/login",
+  Profile: "/profile",
+  restaurantPage: "/restaurant-page",
```

Comment:
Não está seguindo o padrão das outras rotas:

```suggestion
  restaurantDetails: "/restaurant",
```